### PR TITLE
[doc] Improve Pulsar Security-token-admin

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Security document (security-token-admin section): http://pulsar.apache.org/docs/en/next/security-token-admin/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
@Jennifer88huang please review this~

**Comment by Jennifer88huang** [comment]
Ok, got it, will review it later.

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

File: site2/docs/security-token-admin.md [hunk 1]
```diff
@@ -6,96 +6,87 @@ sidebar_label: Token authentication admin
...
-[JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
+Pulsar supports authenticating clients using security tokens that are based on [JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
 
-Tokens are used to identify a Pulsar client and associate with some "principal" (or "role") which
-will be then granted permissions to do some actions (eg: publish or consume from a topic).
+You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that
+is permitted to do some actions (for example, publish to a topic or consume from a topic).
 
-A user will typically be given a token string by an administrator (or some automated service).
+A user typically gets a user a token string from the administrator (or some automated service).
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: check this sentence.

File: site2/docs/security-token-admin.md [hunk 2]
```diff
@@ -6,96 +6,87 @@ sidebar_label: Token authentication admin
...
-The compact representation of a signed JWT is a string that looks like:
+The compact representation of a signed JWT is a string that looks like as the follwing:
 
 ```
- eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
- ```
+eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
+```
+
+Application specifies the token when you are creating the client instance. An alternative is to pass a "token supplier" (a function that returns the token when the client library needs one).
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: Check the first sentence, and pay attention to two points:
1. the word "specifies"
2. use present tense.

File: site2/docs/security-token-admin.md [hunk 3]
```diff
@@ -6,96 +6,87 @@ sidebar_label: Token authentication admin
...
-Application will specify the token when creating the client instance. An alternative is to pass
-a "token supplier", that is to say a function that returns the token when the client library
-will need one.
 
 > #### Always use TLS transport encryption
-> Sending a token is equivalent to sending a password over the wire. It is strongly recommended to
-> always use TLS encryption when talking to the Pulsar service. See
-> [Transport Encryption using TLS](security-tls-transport.md)
+> Sending a token is equivalent to sending a password over the wire. You had better
+> use TLS encryption all the time when you are talking to the Pulsar service. See
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: replace "are talking to" to other words.

File: site2/docs/security-token-admin.md [hunk 4]
```diff
@@ -6,96 +6,87 @@ sidebar_label: Token authentication admin
...
+> [Transport Encryption using TLS](security-tls-transport.md) for more details.
 
 ## Secret vs Public/Private keys
 
 JWT support two different kind of keys in order to generate and validate the tokens:
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: JWT supports...two...kinds...

File: site2/docs/security-token-admin.md [hunk 5]
```diff
@@ -6,96 +6,87 @@ sidebar_label: Token authentication admin
...
+ * Asymmetric: A pair of keys consist of the Private key and the Public key.
+    - You can use ***Private*** key to generate tokens
+    - You can use ***Public*** key to validate tokens
 
 ### Secret key
 
-When using a secret key, the administrator will create the key and he will
-use it to generate the client tokens. This key will be also configured to
-the brokers to allow them to validate the clients.
+When you are using a secret key, the administrator creates the key and uses the key to generate the client tokens. You can also configure this key to the brokers in order to allow the brokers validating the clients.
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: Use present tense, for example, "when you use a xxx". Check all similar cases.

File: site2/docs/security-token-admin.md [hunk 6]
```diff
@@ -105,21 +96,19 @@ $ bin/pulsar tokens create --secret-key file:///path/to/my-secret.key \
 
 ## Authorization
 
-The token itself doesn't have any permission associated. That will be determined by the
-authorization engine. Once the token is created, one can grant permission for this token to do certain
-actions. Eg. :
+The token itself does not have any permission associated. The authorization engine determines whether the token should have permissions or not. Once you have created the token, one can grant permission for this token to do certain actions. For example:
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]:
```suggestion
The token does not associate any permission. The authorization engine determines whether the token has permissions or not. Once you have created the token, you can grant permission for this token to do certain actions. For example:
```

File: site2/docs/security-token-admin.md [hunk 7]
```diff
@@ -105,21 +96,19 @@ $ bin/pulsar tokens create --secret-key file:///path/to/my-secret.key \
...
 
 ```shell
 $ bin/pulsar-admin namespaces grant-permission my-tenant/my-namespace \
             --role test-user \
             --actions produce,consume
 ```
 
-## Enabling Token Authentication ...
+## Enable Token Authentication ...
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: Refine the titles, you can refer to https://github.com/apache/pulsar/pull/5038

File: site2/docs/security-token-admin.md [hunk 8]
```diff
@@ -8,38 +8,37 @@ sidebar_label: Token authentication admin
 
 Pulsar supports authenticating clients using security tokens that are based on [JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
 
-You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that
-is permitted to do some actions (for example, publish to a topic or consume from a topic).
+You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that is permitted to do some actions (for example, publish to a topic or consume from a topic).
 
-A user typically gets a user a token string from the administrator (or some automated service).
+A user typically gives a user a token string from the administrator (or some automated service).
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: A user typically gets a token string from the administrator (or some automated service).

File: site2/docs/security-token-admin.md [hunk 9]
```diff
@@ -96,15 +95,15 @@ $ bin/pulsar tokens create --secret-key file:///path/to/my-secret.key \
...
+The token does not associate any permission. The authorization engine determines whether the token has permissions or not. Once you have created the token, you can grant permission for this token to do certain actions. For example:
 
 ```shell
 $ bin/pulsar-admin namespaces grant-permission my-tenant/my-namespace \
             --role test-user \
             --actions produce,consume
 ```
 
-## Enable Token Authentication ...
+## Enable token authentication
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: Refine it further.

File: site2/docs/security-token-admin.md [hunk 10]
```diff
@@ -8,38 +8,37 @@ sidebar_label: Token authentication admin
...
+A user typically gives a user a token string from the administrator (or some automated service).
 
 The compact representation of a signed JWT is a string that looks like as the follwing:
 
 ```
 eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
 ```
 
-Application specifies the token when you are creating the client instance. An alternative is to pass a "token supplier" (a function that returns the token when the client library needs one).
+Application does the token specification when you create the client instance. An alternative is to pass a "token supplier" (a function that returns the token when the client library needs one).
```
Comment (Jennifer88huang) [inline site2/docs/security-token-admin.md]: re-write this sentence. 
The token is specified by application when you create a client instance.

**Comment by Monica-zy** [comment]
@Jennifer88huang Please check if there are some other problems to be improved. thanks~

**Comment by Jennifer88huang** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by sijie** [comment]
run java8 tests
run integration tests

**Comment by Jennifer88huang** [comment]
run integration tests

**Pull request merged**